BUILD

```diff
@@ -181,6 +181,13 @@ cc_library(
     deps = [":protobuf_lite"],
 )
 
+cc_library(
+    name = "protobuf_headers",
+    hdrs = glob(["src/**/*.h"]),
+    includes = ["src/"],
+    visibility = ["//visibility:public"],
```

Comment:
Please add a comment to explain what this rule is for and advise users to stay away from it. Also add a TODO that we should remove this rule when bazel supports depending on a cc_library without linking.